src/scss/colorways.scss

```diff
@@ -467,6 +467,50 @@
 .gmk-dolch-kb {
 }
 
+.gmk-dualshot-key {
+  background: $color-gmk-abs-2B;
+  color: $color-gmk-abs-N9;
+}
+.gmk-dualshot-mod {
+  background: $color-gmk-abs-2B;
+  color: $color-gmk-abs-N9;
+}
+.gmk-dualshot-mod2 {
+  background: $color-gmk-abs-2B;
+  color: $color-gmk-abs-RO2;
+}
+.gmk-dualshot-mod3 {
+  background: $color-gmk-abs-2B;
+  color: $color-gmk-abs-N6;
+}
+.gmk-dualshot-mod4 {
+  background: $color-gmk-abs-2B;
+  color: $color-gmk-abs-TU2;
+}
+.gmk-dualshot-mod5 {
+  background: $color-gmk-abs-2B;
+  color: $color-gmk-abs-N5;
```

Comment:
K, here's where I've landed on this:

![image](https://user-images.githubusercontent.com/18669334/95026262-f7cecc80-0644-11eb-9732-62313333c692.png)

```suggestion
  background: darken($color-gmk-abs-2B, 5%);
  color: darken($color-gmk-abs-N9, 12%);
}
.gmk-dualshot-mod {
  background: darken($color-gmk-abs-2B, 5%);
  color: darken($color-gmk-abs-N9, 12%);
}
.gmk-dualshot-mod2 {
  background: darken($color-gmk-abs-2B, 5%);
  color: lighten($color-gmk-abs-RO2, 35%);
}
.gmk-dualshot-mod3 {
  background: darken($color-gmk-abs-2B, 5%);
  color: lighten($color-gmk-abs-N6, 5%);
}
.gmk-dualshot-mod4 {
  background: darken($color-gmk-abs-2B, 5%);
  color: lighten($color-gmk-abs-TU2, 9%);
}
.gmk-dualshot-mod5 {
  background: darken($color-gmk-abs-2B, 5%);
  color: lighten($color-gmk-abs-N5, 29%);
```

Using Sass to modify the colors because it leaves the `$color-gmk-abs-*` variables unmodified in case future colorways use those colors.

Reply:
Done.

Reply:
I think the color of `mod2` not close to the GMK Dualshot @noroadsleft. It's like a pink color, not red.

Reply:
I realize this – it's actually the change I dislike the most from this suggestion – but at the font sizes we're dealing with it takes a big shift to get contrast with the base color (medium gray is one of the worst colors to deal with as far as contrast).